lib/msal-common/rollup.config.js

```diff
@@ -26,7 +26,7 @@ export default [
             {
                 file: "./lib/msal-common.js",
                 format: "umd",
-                name: "msal",
+                name: "msal-common",
```

Comment:
Should these match the npm package name `@azure/msal-common`?

Reply:
this is the name of the output from rollup (i.e. in the sample when you call `import as xyz`), I wanted to change it to differentiate between msal-browser output.

Reply:
Right...we saw issue in `msal` where there was a mismatch between the `umd` name and the package name.